admin_guide/sdn_troubleshooting.adoc

```diff
@@ -578,14 +578,11 @@ replaced with an `iptables`-only solution.
 
 == Further Help
 
-. Run the script at
-https://raw.githubusercontent.com/openshift/openshift-sdn/master/hack/debug.sh
-on the master (or from another machine with access to the master) to
-generate useful debugging information.
-. When debugging IP failover problems, run the script at
-https://raw.githubusercontent.com/openshift/openshift-sdn/master/hack/ipf-debug.sh
-on the master (or from another machine with access to the master) to
-generate useful debugging information.
+Run the following to diagnose common network issues:
```

Comment:
It's probably worth saying in what context to run this (basically that you have to run it with the `admin.kubeconfig`). Or possibly a link to more general instructions for running the diagnostics.